appveyor.ps1

```diff
@@ -0,0 +1,35 @@
+# the version under development, update after a release
+$version = '3.0.0'
+$modifier = '-beta-6'
```

Comment:
Clearly, we can change the name. Not the issue for me though.

On Mon, Oct 26, 2015 at 8:46 AM, Rob Prouse <notifications@github.com>
wrote:

> In appveyor.ps1
> <https://github.com/nunit/nunit/pull/932#discussion_r43010445>:
>
> > @@ -0,0 +1,35 @@
> > +# the version under development, update after a release
> > +$version = '3.0.0'
> > +$modifier = '-beta-6'
>
> Switch to -Dev?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/932/files#r43010445>.
>